pylate/models/colbert.py

```diff
@@ -217,14 +214,15 @@ def __init__(
         model_kwargs: dict | None = None,
         tokenizer_kwargs: dict | None = None,
         config_kwargs: dict | None = None,
-        model_card_data: Optional[SentenceTransformerModelCardData] = None,
+        model_card_data: Optional[PylateModelCardData] = None,
```

Comment:
you can use  `PylateModelCardData | None = None` here for consistency

Reply:
Done